templates/project-ts/src/Add.test.ts

```diff
@@ -20,7 +21,10 @@ describe('Add', () => {
     zkApp: Add;
 
   beforeAll(async () => {
-    if (proofsEnabled) await Add.compile();
+    if (proofsEnabled) {
```

Comment:
This will handle proofs enabled or disabled:

```ts
    await AddZKprogram.compile({ proofsEnabled });
    if (proofsEnabled) {
      await Add.compile();
    }
```

Reply:
Good call.